packages/@okta/vuepress-site/docs/reference/core-okta-api/index.md

```diff
@@ -26,30 +26,321 @@ REST endpoints to configure objects whenever you need. For example:
+
+> **Note:** All `Date` values use the ISO 8601 format `YYYY-MM-DDTHH:mm:ss.SSSZ`.
+
+#### Attribute operators
+
+| Operator | Description | Behavior                                                         |
+| -------- | ----------- | --------                                                         |
+| `and`      | Logical AND | The filter is only a match if both expressions evaluate to true. |
+| `not` | Logical NOT | The filter is a match if the expression evaluates to false. | 
+| `or`       | Logical OR  | The filter is a match if either expression evaluates to true.    |
+
+#### Logical operators
+
+| Operator | Description         | Behavior                                |
+| -------- | -----------         | --------                                                                                   |
+| `()`       | Precedence grouping | Boolean expressions may be grouped using parentheses to change the standard order of operations: for example, evaluate OR logical operators before logical AND operators. |
+
+Filters must be evaluated using standard order of operations. Attribute operators have the highest precedence, followed by the grouping operator (for example, parentheses), followed by the logical `AND` operator, followed by the logical `OR` operator.
+
+### Hypermedia
+
+Objects in the Okta API use hypermedia for discoverability. Hypermedia enables API clients to navigate objects by following links like a web browser instead of hard-coding URLs in your application. Links are identified by link relations that are named keys. Link relations describe what objects are available and how API clients can interact with them. Each object may publish a set of link relationships based on the state of the object. For example, the status of a user in the [User API](/docs/reference/api/users/#links-object)  governs which lifecycle operations are permitted. Only the permitted operations are published as lifecycle operations.
+
+The Okta API incorporates [JSON Hypertext Application Language](http://tools.ietf.org/html/draft-kelly-json-hal-06) or HAL format as the foundation for hypermedia discoverability. HAL provides a set of conventions for expressing hyperlinks in JSON responses that represent two simple concepts: Resources and Links.
+
+> **Note:** The HAL-specific media type `application/hal+json` isn't currently supported as a formal media type for content negotiation. Use the standard `application/json` media type. As we get more experience with the media format, we may add support for the media type.
+
+### Links
+
+Object whose property names are link relation types (as defined by [RFC8288](http://tools.ietf.org/html/rfc8288)) and values are either a Link object or an array of Link objects.
```

Comment:
The first part of the sentence doesn't make sense:
"Object whose property names are..."

I would avoid using "whose" here ("whose" is typically used for  people).